src/flexmock/_api.py

```diff
@@ -1315,25 +1315,20 @@ def get_flexmock_expectation(
             args = {"kargs": args, "kwargs": {}}
         if not isinstance(args["kargs"], tuple):
             args["kargs"] = (args["kargs"],)
-        if name and obj in cls.flexmock_objects:
```

Comment:
I guess we can have `obj not it cls.flexmock_objects` if that classmothod is called directly by the user; should it become a private method ?

Reply:
`FlexmockContainer` itself is behind private module `_api` so I don't think we need to mark these methods private.